packages/grid/_modules_/grid/hooks/features/useColumnResize.tsx

```diff
@@ -110,12 +111,134 @@ export const useColumnResize = (columnsRef: React.RefObject<HTMLDivElement>, api
     doc.addEventListener('mouseup', handleResizeMouseUp);
   });
 
+  // TODO: remove support for Safari < 13.
+  // https://caniuse.com/#search=touch-action
+  //
+  // Safari, on iOS, supports touch action since v13.
+  // Over 80% of the iOS phones are compatible
+  // in August 2020.
+  let cachedSupportsTouchActionNone;
+  function doesSupportTouchActionNone() {
+    if (cachedSupportsTouchActionNone === undefined) {
+      const element = document.createElement('div');
+      element.style.touchAction = 'none';
+      document.body.appendChild(element);
+      cachedSupportsTouchActionNone = window.getComputedStyle(element).touchAction === 'none';
+      element.parentElement!.removeChild(element);
+    }
+    return cachedSupportsTouchActionNone;
+  }
+
+  function trackFinger(event, currentTouchId) {
```

Comment:
Same here

Reply:
Same as the discussion above, I'll change it to a memo and make it more optimal.

Reply:
I don't get it, why does it need a memo? I can be in the outer scope, it doesn't need to be in the scope of the hook.